docs/setup/settings.asciidoc

```diff
@@ -128,6 +128,14 @@ store saved searches, visualizations and dashboards. Kibana creates a new index
 if the index doesn’t already exist. If you configure a custom index, the name must
 be lowercase, and conform to {es} {ref}/indices-create-index.html[index name limitations].
 
+`kibana.autocompleteTimeout:`:: *Default: "1000"* Time in milliseconds to wait
+for autocomplete suggestions from Elasticsearch. This value must be a positive
+integer excluding zero.
```

Comment:
@flash1293 By definition, a positive integer is a whole number greater than zero.  So I'd either:

- Remove "excluding zero"
- Rewrite this sentence as "This value must be a whole number greater than zero."